src/assets/svg/player-series-record.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let us remove width="40" height="40". In that way the icon will scale to the parent container and we don't need to override it with CSS.